saleor/order/utils.py

```diff
@@ -438,6 +440,7 @@ def update_gift_card_balance(
         total_price_left = total_price_left - gift_card.current_balance
         gift_card.current_balance_amount = 0
     balance_data.append((gift_card, previous_balance.amount))
+    return total_price_left
```

Comment:
We can update the typing in the signature now when the function returns a value.